src/components/CommentSection.jsx

```diff
@@ -1,22 +1,39 @@
-import { useEffect } from "react"
+import { useContext, useEffect } from "react"
 import { useState } from "react"
-import { getReviewComments } from "./api"
+import { getReviewComments, postComment } from "./api"
 import { useParams } from "react-router-dom"
 import CommentCard from "./CommentCard"
 import { Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from "@mui/material"
+import { UserContext } from "../contexts/user"
 
 
 
 function CommentSection (){
+    const [commentInput, setCommentInput] = useState("")
     const {reviewID} = useParams()
     const [reviewComments, setReviewComments] = useState([])
     const [isLoading, setIsLoading] = useState(true)
     const [showComments, setShowcomments] = useState(false)
-
+    const { username } = useContext(UserContext)
+    const [commentErr, setCommentErr] = useState(null)
+    
     const commentHandler = (event) => {
         event.preventDefault()
         setShowcomments(!showComments)
     }
+    const postCommentHandler = (event) => {
+        event.preventDefault()
+        if(commentInput.length > 10){
+            postComment(reviewID,{username: username.username, body: commentInput})
```

Comment:
This line is what makes me think the user context should have a state of 'user' not 'username'. It's an object containing the property of username